ingest-app/docs/api.md

```diff
@@ -666,6 +667,8 @@ Example: Initiate a bulk update of 3 collections. Find platforms that have Type
 ```
 curl -i -XPOST -H "Cmr-Pretty:true" -H "Content-Type: application/json" -H "Echo-Token: XXXX" %CMR-ENDPOINT%/providers/PROV1/bulk-update/collections -d
 '{"concept-ids": ["C1200000005-PROV1","C1200000006-PROV1","C1200000007-PROV1"],
+  "name": "TEST NAME",
+  "create-at": "2017-10-24T17:00:03.000Z",
```

Comment:
Did you meant to add created-at?  Created-at is automatically populated in the database table. Why is it included in the request?

Reply:
This probably came from a (wrong) comment from me. I thought this was a JSON response, didn't really this was a request body. Agree that it should not be in the request.